exchanges/huobi/huobi_test.go

```diff
@@ -1576,13 +1576,13 @@ func TestGetHistoricCandles(t *testing.T) {
 		t.Error(err)
 	}
 
-	startTime := time.Now().Add(-time.Hour * 1)
-	_, err = h.GetHistoricCandles(context.Background(), pair, asset.Spot, kline.OneMin, startTime, time.Now())
+	endTime := time.Now().Add(-time.Hour).Truncate(time.Hour)
+	_, err = h.GetHistoricCandles(context.Background(), pair, asset.Spot, kline.OneMin, endTime.Add(-time.Hour), endTime)
```

Comment:
I have had a play around with your changes here. The only issue I can see with changes coming from offsetting the time is that we are in fact losing trading volume; for example if I create a 24 hour candle from the one hour candles which will be UTC aligned, it will be a better approximation of what actually occurred.  It seems like OneWeek, OneMonth, and OneYear intervals are all offset to shanghai time as well. 

This is a doozy 😆. WRT week and higher candles 8 hr drift of data doesn't seem to be a huge issue (or could be depending on algo 🤷) so maybe remove the allowance of OneDay interval in wrapper and then construct it from the hourly candles and then leave a comment then offset the higher values maybe?

Reply:
I removed the time padding and changed the candles to up-convert from hourly candles. I understand you're working on the same/similar issue in your PR atm